src/GraphQL/Execution/ExecutionContext.cs

```diff
@@ -110,5 +110,12 @@ protected virtual void ClearContext()
                 _trackedArrays.Clear();
             }
         }
+
+        /// <summary>
+        /// Provides a method for an execution strategy to reuse an instance of <see cref="ReadonlyResolveFieldContext"/>.
+        /// Although in theory this field should not be accessed by multiple threads at the same time,
```

Comment:
Why should not ? What about `ParallelExecutionStrategy` and async resolvers?

Reply:
Hmm......actually, you're right.  Multiple resolvers can run in parallel, and a resolver can run in parallel with ExecutionStrategy, and the code after the await within ExecuteNodeAsync could execute at the same time as other code within ExecutionStrategy.  So it actually is necessary.

Reply:
Comment updated.